setup/apache.yml

```diff
@@ -18,7 +18,7 @@ apache_cosign_deny_friend:     "global=app_cosign_deny_friend default=no"
 
 apache_static_path:
   global: deploy_root
-  output: File.join(deploy_root, app_name, "shared", "public")
+  output: File.join(deploy_root, app_name, "current", "public")
```

Comment:
Or more accurately (as explained to me by @sethaj) - the jekyll templates are in the same repository, and when they deploy heliotrope, capistrano runs a rake task in `current` to rebuild the jekyll site into `current/public`.